AWSO 2.8.0 changes (#3933)
SumoLogic/sumologic-documentation: docs/observability/aws/about.md

The following Sumo Logic [AWS Observability Solution apps](/docs/observability/aws/integrations) provide the ability to quickly isolate and solve problems with their specialized pre-configured dashboards:

* [AWS Observability API Gateway](/docs/observability/aws/integrations/aws-api-gateway). The [Amazon API Gateway](https://aws.amazon.com/api-gateway/) service allows you to create RESTful APIs and WebSocket APIs for real-time two-way communication applications in containerized and serverless environments, as well as web applications. <br/> Sumo Logic's AWS Observability API Gateway dashboards provide insights into Amazon API Gateway tasks while accepting and processing concurrent API calls throughout your infrastructure, including traffic management, CORS support, authorization, access control, throttling, monitoring, and API version management.

> * [AWS Observability Application Load Balancer](/docs/observability/aws/integrations/aws-application-load-balancer). The [AWS Application Load Balancer](https://aws.amazon.com/elasticloadbalancing/) service functions at the application layer, it receives requests, evaluates the listener rules in priority order to determine which rule to apply, and then selects a target from the target group.<br/> Sumo Logic's AWS Elastic Load Balancing app is a unified logs and metrics app that gives you visibility into the health of your AWS Application Load Balancer and target groups. This app's preconfigured dashboards provide insights into latency, request and host status, threat intel, and HTTP backend codes by availability zone and target group.

Reviewer: ```suggestion
* [AWS Observability Application Load Balancer](/docs/observability/aws/integrations/aws-application-load-balancer). The [AWS Application Load Balancer](https://aws.amazon.com/elasticloadbalancing/) service functions at the application layer. It receives requests, evaluates the listener rules in priority order to determine which rule to apply, and then selects a target from the target group.<br/> Sumo Logic's AWS Elastic Load Balancing app is a unified logs and metrics app that gives you visibility into the health of your AWS Application Load Balancer and target groups. This app's preconfigured dashboards provide insights into latency, request and host status, threat intel, and HTTP backend codes by availability zone and target group.
```

* [Amazon ElastiCache](/docs/observability/aws/integrations/amazon-elasticache). The [Amazon ElastiCache](https://aws.amazon.com/elasticache/) allows you to set up, run, and scale popular open-source compatible in-memory data stores in the cloud.<br/>The Amazon ElastiCache dashboards provide visibility into key event and performance analytics that enable proactive diagnosis and response to system and environment issues. Use the preconfigured dashboards for at-a-glance analysis of event status trends, locations, successes and failures, as well as system health and performance metrics. The dashboards also have additional performance insights for Redis clusters.

> * [AWS Network Load Balancer](/docs/observability/aws/integrations/aws-network-load-balancer). The [AWS Network Load Balancer](https://aws.amazon.com/elasticloadbalancing/network-load-balancer/) service is distributed in OSI Layer 4 (i.e., network) traffic (e.g., TCP, UDP, TLS) and can handle over a million requests per second.<br/>The AWS Network Load Balancer dashboards provide insights to ensure that your network load-balancers are operating as expected, backend hosts are healthy and to quickly identify errors.

Reviewer: ```suggestion
* [AWS Network Load Balancer](/docs/observability/aws/integrations/aws-network-load-balancer). The [AWS Network Load Balancer](https://aws.amazon.com/elasticloadbalancing/network-load-balancer/) service is distributed in OSI Layer 4 (that is, network) traffic (for example, TCP, UDP, TLS) and can handle over a million requests per second.<br/>The AWS Network Load Balancer dashboards provide insights to ensure that your network load-balancers are operating as expected, backend hosts are healthy and to quickly identify errors.
```
